packages/plugins/model-define/tfjs-simplecnn-model-define/src/index.ts

```diff
@@ -47,8 +47,8 @@ const simpleCnnModelDefine: ModelDefineType = async (data: ImageDataset, args: M
     outputShape = Object.keys(data.metadata.labelMap).length;
     labelMap = data.metadata.labelMap;
   } else {
-    const log = JSON.parse(fs.readFileSync(path.join(recoverPath, 'metadata.json'), 'utf8'));
-    labelMap = log.output.dataset.labelMap;
+    const log = JSON.parse(fs.readFileSync(path.join(recoverPath, '..', 'metadata.json'), 'utf8'));
```

Comment:
I'm not sure if we should read the metadata.json in a plugin, how about having some methods to `metadata.json` in "@pipcook/pipcook-core"? So that plugin maintainer will not care about how the metadata.json is.

Never mind, this change LGTM in this PR.